Js/Pop.js

```diff
@@ -0,0 +1,118 @@
+const Modalh = document.querySelector('.Modalh');
+const Modalrole = document.querySelector('.Modalrole');
+const Modalyear = document.querySelector('.Modalyear');
+const ModalImg = document.querySelector('.ModalImg');
+const Modalpara = document.querySelector('.Modalpara');
+const ModalWorks = document.querySelector('.ModalWorks');
+const Modalcan = document.querySelector('.Modalcan');
+const cross = document.querySelector('.cross');
+const cardCover = document.querySelector('.cardCover');
+
+const Arrayprojects = [
+  {
+    title: 'Tonic',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    imageLink: './images/Snapshoot-Portfolio-(3).jpg',
+    tags: ['HTML', 'CSS', 'Javascript'],
+    role: 'Back End Dev',
+    company: 'CANOPY',
+    year: '2015',
+    sourceLink: '',
+    demoLink: '',
+  },
+  {
+    title: 'Mulit-Post Stories',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    imageLink: './images/Snapshoot-Portfolio(4).jpg',
+    tags: ['HTML', 'CSS', 'Javascript'],
+    role: 'Back End Dev',
+    company: 'CANOPY',
+    year: '2015',
+    sourceLink: '',
+    demoLink: '',
+  },
+  {
+    title: 'Tonic',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    imageLink: './images/Snapshoot-Portfolio(5).jpg',
+    tags: ['HTML', 'CSS', 'Javascript'],
+    role: 'Back End Dev',
+    company: 'CANOPY',
+    year: '2015',
+    sourceLink: '',
+    demoLink: '',
+  },
+  {
+    title: 'Mulit-Post Stories',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    imageLink: './images/Snapshoot-Portfolio(6).jpg',
+    tags: ['HTML', 'CSS', 'Javascript'],
+    role: 'Back End Dev',
+    company: 'CANOPY',
+    year: '2015',
+    sourceLink: '',
+    demoLink: '',
+  },
+];
```

Comment:
Both the model and popup should be generated from the same array of projects. If you see, `Arrayprojects` is just a copy of the another array you already have `Model.js`. It would make more sense if you used the same array for popups.